### ISSUE TITLE
CI-windows fails

### ISSUE THREAD

**Issue by pz-max** [pz-max/pypsa-africa#266]
## Describe the Bug

*Please provide a description of what the bug is and add a minimal example/command for reproducing the bug.*
We splitted the CI's into Windows and Linux/Max. The GitHub actions workflows can be found [here](https://github.com/pypsa-meets-africa/pypsa-africa/tree/main/.github/workflows).

When clicking on the CI windows badge 
![image](https://user-images.githubusercontent.com/61968949/154255213-2fcc5e4b-d442-4988-bfb5-2f298ee4a86e.png) and analysing the error message, Windows seems to have an issue with the solver 'ipopt' usage in the cluster network

## Error Message

```bash
pyomo.common.errors.ApplicationError: No executable found for solver 'ipopt'
```

Full log: https://github.com/pypsa-meets-africa/pypsa-africa/runs/5195198259?check_suite_focus=true

**Comment by davide-f** [comment]
I think I solved the problem: it is a matter of ipopt version.
Once the #260 is merged, I'll fix this issue; it should be fast to do

**Comment by davide-f** [comment]
Closed as solved by #271
